projects/packages/sync/src/modules/class-plugins.php

```diff
@@ -379,4 +385,21 @@ public function expand_plugin_data( $args ) {
 			$plugin_data,
 		);
 	}
+
+	/**
+	 * Helper method for firing the 'jetpack_plugins_updated' action on shutdown.
+	 */
```

Comment:
I wonder if it's worth adding the `@access` docblock to the new `sync_plugins_updated` function, for consistency with the rest of the functions more than anything else.

```suggestion
	 * Helper method for firing the 'jetpack_plugins_updated' action on shutdown.
	 *
	 * @access public
	 */
```